tests/test_anyflowmodel.py

```
@@ -0,0 +1,591 @@
+
+    cfg = mod.create_config()
+    shift = float(cfg.model.sample_t_cfg.shift)
+    n = int(cfg.model.student_sample_steps)
+    max_t = float(cfg.model.net.max_t)  # the bound `_rollout_t_list` clamps to
+
+    grid = [1.0 - i / n for i in range(n + 1)]
+    expected = [min(shift * x / (1 + (shift - 1) * x), max_t) for x in grid]
+
+    t_list = list(cfg.model.sample_t_cfg.t_list)
+    assert len(t_list) == n + 1, t_list
+    assert all(abs(a - b) < 1e-9 for a, b in zip(t_list, expected)), (t_list, expected)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)